mlflow/pipelines/steps/split.py

```diff
@@ -202,7 +202,7 @@ def _run(self, output_directory):
                 importlib.import_module(post_split_module_name), post_split_fn_name
             )
             (train_df, validation_df, test_df) = post_split(train_df, validation_df, test_df)
-
+            _logger.info(f"Running {post_split_fn_name} on train_df, validation_df, and test_df.")
```

Comment:
Do we want to change this to `Ran {post_split_fn_name} on train_df, validation_df, and test_df.` since its technically after we ran it?

Or move this logger before we run it?

This way its easier to debug issues as well. Like if a users sees `running {post_split_fn_name} on train_df, validation_df, and test_df.` and sees some error, it might indicate that `post_split_fn_name` is where something is broken. What do you think?

Reply:
Good call out. Let's make this "running..." and happen before. This is more consistent with other logging outputs.

Reply:
Are `train_df`, etc user-facing names? If not, let's say something like "Running ... on train, validation and test datasets"

Reply:
@apurva-koti good point, i'm not seeing other logging that mentions those dfs explicitly. updated the message as you suggested.